railties/lib/rails/engine/configuration.rb

```diff
@@ -88,9 +87,11 @@ def autoload_paths
       end
 
       def webpacker_path
-        if File.file?("#{Rails.root}/config/webpacker.yml")
-          YAML.load_file("#{Rails.root}/config/webpacker.yml")[Rails.env]["source_path"]&.gsub("app/", "")
-        else
+        @webpacker_path ||= begin
+          return "javascript" unless File.file?("#{Rails.root}/config/webpacker.yml")
+          require "webpacker"
```

Comment:
Rails engine can't depend on webpacker. Why we need to load webpacker now?

Reply:
it was originally reading and parsing yaml.  instead of depending on that, i used webpacker based on a suggestion from @y-yagi to get the data.  if webpacker doesn't load, we return the default.

Reply:
Webpacker fallbacks to `production` env if current env is not defined in `webpacker.yml`, and we need the same behavior here.   

I think it is better to use Webpacker's methods here than to emulate such Webpacker-specific behavior. (Emulating the behavior will make it difficult to cope with changes in Webpacker specifications.)

Reply:
In that case we should invert this dependency. Webpacker should set the path in their railtie and not having rails depending on webpacker.

Reply:
alright, I think I've got it sorted out.  I'll push up the changes here and create a PR in the Webpacker repo as well.